kubectl-rook-ceph.sh

```diff
@@ -32,8 +32,16 @@ function print_usage () {
   echo "COMMANDS"
   echo "  help           : output help text"
   echo "  ceph <args>    : call a 'ceph' CLI command with arbitrary args"
+  echo "  rbd <args>     : Call a 'rbd' CLI command with arbitrary args"
+  echo "  mon-endpoints  : output mon endpoints"
```

Comment:
I vote for `mons`. Are there votes to keep `mon-endpoints`? Let's discuss in huddle.
```suggestion
  echo "  mons           : output mon endpoints"
```